content/docs/user-guide/index.md

```diff
@@ -1,51 +1,81 @@
-# Overview
+# What is DVC?
+
+<!--
+## What is DVC?
+-->
+
+**Data Version Control** is a [free], open-source tool for [data management],
+[ML pipeline][ml pipelines] automation, and [experiment management]. This helps
+data science and machine learning teams manage **large datasets**, make projects
+**reproducible**, and **collaborate** better.
+
+DVC takes advantage of the existing software engineering toolset your team
+already knows (Git, your IDE, CI/CD, etc.). Its design follows this set of
```

Comment:
Maybe add "your storage"?

Reply:
Added "cloud storage" to the examples list.